docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,264 @@ include::includes/global-attributes.asciidoc[]
+In _Definition_ ::
+* _IP Address/MAC Address/Range (CIDR)_, give the network address of your _Production_ network; For us, it will be *172.16.1.0/24*
+* _Description_, give a description so you can quickly see what it is about
+* From the _Type_ list, select *Aerohive AP*
+* From _Mode_ select *Production*
+* _Switch Group_ by default set to *None*
+* _Deauthentication Method_ set to *RADIUS*
+* Click *SAVE*
 
-image::docs/images/aerohive-radius-pf.png[scaledwidth="100%",alt="RADIUS server configuration"]
+image::docs/images/pfaerohive10.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
-Click Ok to save your changes.
 
-Configure the Open SSID
-+++++++++++++++++++++++
-On the left menu, click on 'SSIDs'. Click the 'New' button, and configure the SSID like in the following screen capture:
+'''
 
-image::docs/images/aerohive-ssid-public.png[scaledwidth="100%",alt="SSID open configuration"]
 
-Click OK to save the SSID.
+In _Role_ ::
+* Set all VLans ID in each roles
 
-Configure the Secure SSID
-+++++++++++++++++++++++++
-On the left menu, click on 'SSIDs'. Click the 'New' button, and configure the SSID like in the following screen capture:
+image::docs/images/pfaerohive11.png[scaledwidth="100%",alt="New switch configuration",align=right]
 
-image::docs/images/aerohive-ssid-secure.png[scaledwidth="100%",alt="SSID secure configuration"]
 
-Click OK to save the SSID.
+'''
 
-Configure User Profile
-^^^^^^^^^^^^^^^^^^^^^^
-You will now need to create user profile matching your VLAN assignments, the "Default VLAN" have been created before to match the desired VLAN number.
 
-Create user profile as follow:
+In _RADIUS_ ::
+* Modify the secret passphrase previously sets in the Aerohive HiveManager
+* Click on *SAVE*
 
-image::docs/images/aerohive-user-profile-reg.png[scaledwidth="100%",alt="User profile registration"]
+This ends the PacketFence configuration
 
-image::docs/images/aerohive-user-profile-prod.png[scaledwidth="100%",alt="User profile production"]
+'''
 
-Configure SNMP
-^^^^^^^^^^^^^^
-You now need to configure the SNMP to add a valid read community string. Click on the 'Configure' pane, and on 'Advanced Configuration', then 'Management Policies', then 'SNMP Assignments'. Now click on 'New' and follow this configuration.
+Web Auth Configuration
+----------------------
 
-image::docs/images/aerohive-add-snmp.png[scaledwidth="100%",alt="SNMP configuration"]
+In this part we will describe the WebAuth configuration for Aerohive APs
 
-<<<
 
-WebAuth Enforcement
-^^^^^^^^^^^^^^^^^^^
-Setup the SSIDs
-^^^^^^^^^^^^^^^
-In this guide, we will configure one open SSID using RADIUS MAC-based filtering and one secured SSID using WPA2 Enterprise.
+Assumption :
+~~~~~~~~~~~~
 
-Login to your HiveManager using the admin credentials. You should see the Dashboard. Click on the 'Configure' pane, and follow the following steps.
+In this part, the default VlAN must be the native VLAN.
```

Comment:
VLAN

Reply:
Not really, it can be another vlan id than the native vlan (In this part, the default VlAN must be the native VLAN.)